client/.storybook/manager.js

```diff
@@ -0,0 +1,6 @@
+import { addons } from "@storybook/addons";
+import { themes } from "@storybook/theming";
+
+addons.setConfig({
+  theme: themes.dark,
+});
```

Comment:
I think there is a new way to add addons like `themes` without `@storybook/addons`.
(didn't test it)

// .storybook/preview.js
```js
import { themes } from '@storybook/theming';

export const parameters = {
  docs: {
    theme: themes.dark,
  },
};
```

See https://storybook.js.org/docs/react/writing-stories/parameters#global-parameters
and 
https://github.com/storybookjs/storybook/blob/next/docs/snippets/common/storybook-preview-docs-dark-theme.js.mdx

Reply:
super its working :)